NosTests/Views/CompactNoteViewTests.swift

```diff
@@ -19,7 +25,6 @@ final class CompactNoteViewTests: CoreDataTestCase {
         fullEvent.contentLinks = [try XCTUnwrap(URL(string: eventContent))]
 
         let subject = CompactNoteView(note: fullEvent)
-        ViewHosting.host(view: subject.environment(\.managedObjectContext, persistenceController.container.viewContext))
```

Comment:
This line was extra. The test does the same thing without it. I did try adding the `.environment(\.managedObjectContext, persistenceController.container.viewContext)` modifier to the next line to see if that fixed the crash but it didn't :(